sources/api/apiclient/src/main.rs

```diff
@@ -223,7 +258,28 @@ fn usage() -> ! {
 
         report cis-k8s options:
             -f, --format               Format of the CIS report (text or json). Default format is text.
-            -l, --level                CIS compliance level to report on (1 or 2). Default is 1."#,
+            -l, --level                CIS compliance level to report on (1 or 2). Default is 1.
+
+        ephemeral-storage init options:
+            -fs, --filesystem          Filesystem to initialize the array as (ext4 or xfs). Default is
```

Comment:
nit: short option should be one character. Maybe also use `-t` in case we ever want `-f` to mean `--force`.
```suggestion
            -t, --filesystem-type      Filesystem to initialize the array as (ext4 or xfs). Default is
```